File: local-data-persistent/LatihanRepository (starter)/java/app/src/main/java/com/dicoding/newsapp/data/remote/retrofit/ApiService.java
package com.dicoding.newsapp.data.remote.retrofit;

import com.dicoding.newsapp.data.remote.response.NewsResponse;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ApiService {
    @GET("top-headlines?country=id&category=science")
    Call<NewsResponse> getNews(@Query("apiKey") String apiKey);
}

File: local-data-persistent/LatihanRepository (coroutines)/app/src/main/java/com/dicoding/newsapp/data/remote/retrofit/ApiService.kt
package com.dicoding.newsapp.data.remote.retrofit

import com.dicoding.newsapp.data.remote.response.NewsResponse
import retrofit2.http.GET
import retrofit2.http.Query

interface ApiService {
    @GET("top-headlines?country=id&category=science")
    suspend fun getNews(@Query("apiKey") apiKey: String): NewsResponse
}